### ISSUE TITLE
body position should keep in the same place when widget is open

### ISSUE THREAD

**Issue by akccakcctw** [freshdesk/freshchat-widget#6]
Demo page here, it's our company site: https://m.kkday.com/en/product/7781

Reproduce steps: 

1. go to demo page
![Snipaste_2019-10-04_18-40-40](https://user-images.githubusercontent.com/16360246/66201918-153a2500-e6d7-11e9-9517-e4a77f1486dc.jpg)

2. scroll down and open freshchat widget (must login, but you can use third party login with facebook or google, doesn't take a long time)
![Snipaste_2019-10-04_18-47-13](https://user-images.githubusercontent.com/16360246/66202061-69dda000-e6d7-11e9-85ba-f9b83818fd6f.jpg)

3. when the freshcat widget opened, body position change to the top
![Snipaste_2019-10-04_18-43-49](https://user-images.githubusercontent.com/16360246/66201932-1f5c2380-e6d7-11e9-922d-dded50aeca3c.jpg)

4. after widget closed, the body position is still at the top

I think that the widget should not change body position, it's not user friendly. Thanks!

**Comment by KingsleyChimezie** [comment]
I agree with @akccakcctw it's not user friendly at all. Can we please get an update on this!
